src/plugins/kibana/public/visualize/editor/agg_params.html

```diff
@@ -10,4 +10,10 @@
     style="display: none;">
 </div>
 
-<!-- schema editors get added down here: aggSelect.html, agg_types/controls/*.html -->
\ No newline at end of file
+<div ng-if="agg.schema.deprecate" class="form-group">
+  <p class="vis-editor-agg-error">
+    "{{ agg.schema.title }}" has been deprecated.
```

Comment:
This is makes it sound like splitting of all charts has been deprecated. Maybe it would make sense to be able to provide a deprecation notice in `src/plugins/kbn_vislib_vis_types/public/tile_map.js`